src/bin/cargo/commands/test.rs

```diff
@@ -38,12 +38,12 @@ pub fn cli() -> Command {
         )
         .arg_targets_all(
             "Test only this package's library unit tests",
-            "Test only the specified binary",
-            "Test all binaries",
+            "Test only the specified binary's unit tests",
```

Comment:
For myself, I would prefer removing references to unit tests, rather than focusing more on unit/integration test terminology.

Its a bit of a misnomer to refer to them as unit/integration tests.  The mechanism allows whitebox / blackbox testing and the user can put unit or integration tests in either location.

Reply:
Agree on Ed's argument. Mentioning unit tests is still confusing. “What does it mean by binary unit tests?”